### DISCUSSION TITLE
[QA] DETAIL

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#262] [Q&A]
DETAIL 페이지 QA를 부탁드려용 
예쁜 말로 둥글둥글 부탁드립니당 〰️

**Comment by YuHyun-P** [comment]
![image](https://user-images.githubusercontent.com/96400112/174761509-810679aa-9373-434e-97a2-93237cf24aa7.png)
서버가 불안정해서 오류인지는 모르겠지만 자신이 작성한 댓글이 알림으로 오네요!

**Reply by hwoo3303** [reply]
엇 이 부분 놓쳤네요. 작성자 본인이 댓글을 달 경우에는 알림 post 안 찌르게 수정했어요~

**Comment by YuHyun-P** [comment]
![url 변경](https://user-images.githubusercontent.com/96400112/174762095-f9321a19-de23-410c-8ffc-accfb501f31a.gif)

상세보기에서 알림으로 뜬 게시글로 이동하면 url은 변경되지만 게시글이 변경되지 않네요!

**Reply by hwoo3303** [reply]
postId 상태 처리를 안해줬네용. 수정했습니다~~

**Comment by Hyevvy** [comment]
이걸 어디다가 리포트할 지 몰라서 우선은 상세페이지랑 프로필페이지 두 군데 모두 적습니다 

상세페이지에서 유저 프로필로는 가지는데 이 때 마이페이지로 이동이 안 돼용 

https://user-images.githubusercontent.com/72402747/174765811-0f9b8526-ac90-400a-9424-9b104f4676f0.mov

**Reply by hwoo3303** [reply]
URL은 변경되는데 페이지는 그대로인걸로 봐서 마이페이지에서 랜더링을 해줘야될거같아요!

**Comment by Hyevvy** [comment]
뒤로가기랑 하트에 `cursor:pointer`를 주는게 좋을 거 같아용 !!

<img width="126" alt="스크린샷 2022-06-21 오후 6 30 38" src="https://user-images.githubusercontent.com/72402747/174767329-2de5c963-cd4f-4917-a76f-3726e644265d.png">

**Reply by hwoo3303** [reply]
좋아여!☺️

**Comment by Hyevvy** [comment]
헙 댓글이 작성되고 나서도 값이 남아있는지 버튼만 눌려도 이전 댓글이 한 번 더 써지네용 

https://user-images.githubusercontent.com/72402747/174768497-d7394d01-4084-427f-8130-226f3ba0f95e.mov

**Reply by hwoo3303** [reply]
감사합니다! value 상태관리를 안해줬었네용

**Comment by hyunjin0910** [comment]
(optional) 
마이페이지에서 내가 쓴글을 봐서 삭제하면 메인으로 돌아가지더라고요. 
유저 경험상 마이페이지 내가 쓴 글로 돌아가는게 편하지 않을까 해서요!
내글을 메인인지, 마이페이지를 통해서 왔는지 확인하는게 어려울 수도 있을 것 같아서 그냥 제안해봅니다 ㅎㅎ
